torch/distributed/rendezvous.py

```diff
@@ -76,9 +76,8 @@ def rendezvous(url: str, rank: int = -1, world_size: int = -1, **kwargs):
         )
         if rank != -1:
             query_dict["rank"] = rank
```

Comment:
@billdarrow My bad, it looks like _query_to_dict returns Dict[str,str]. So these arguments do actually have to be string to pass the linter. We can either update the python type hints or the arguments.

Reply:
No problem, I added back the str() casts...